addons/website/tests/test_snippets.py

```diff
@@ -129,3 +129,6 @@ def test_snippet_image_gallery_thumbnail_update(self):
 
     def test_dropdowns_and_header_hide_on_scroll(self):
         self.start_tour(self.env['website'].get_client_action_url('/'), 'dropdowns_and_header_hide_on_scroll', login='admin')
+
+    def test_snippet_image_crop(self):
+        self.start_tour(self.env['website'].get_client_action_url('/'), 'snippet_image_crop', login='admin')
```

Comment:
![image](https://github.com/user-attachments/assets/aa8e8231-a3a6-4f63-b21e-489ecbd83a0c)

> ** The cases that are breaking are known errors.

It's not because they are marked as such that they are fixed errors... it is just guess from the traceback I guess. Here it occurs during your new test, I don't think it make sense to consider it fixed :grimacing: 

Try your test in local, it does not work. It is making requests to external routes, which a test cannot do, unless those routes are mocked. I don't know by heart how it must be done but I am sure you'll find example in the code :pray:

Reply:
@qsm-odoo Thanks for your review, we will check this.

Reply:
Hello @qsm-odoo regarding the point:
> It is making requests to external routes, which a test cannot do,

Well yes that is because of the [step](https://github.com/odoo/odoo/pull/188577/files#diff-00222fdac794e2247ae5abb055c8670c353cad1c368d45b45809256d49e39450R23-R27) where we are searching for the illustration image.The external request is because of that.
What I will do now is write a continued test for the same scenario i found while digging more about it in test written [here](https://github.com/odoo/odoo/blob/17.0/addons/website/static/tests/tours/media_dialog.js).
Thanks for you valuable insight on this. :star_struck:

Reply:
@qsm-odoo Changes has been done and branch is green, Thanks.